pkgs/top-level/all-packages.nix

```diff
@@ -997,6 +997,8 @@ with pkgs;
 
   edac-utils = callPackage ../os-specific/linux/edac-utils { };
 
+  eduvpn-client = callPackage ../applications/networking/python-eduvpn-client { };
```

Comment:
I think this belongs into `python-packages.nix` as it also provides a Python API: http://python-eduvpn-client.readthedocs.io/en/master/API.html

Reply:
It may have that, but essentially it is really an application people can use to get a VPN account. The API is minimal.

Reply:
So, you don't think the API will be useful in the future?

Reply:
Do they support multiple versions of python? That would be a reason to put it into `pythonPackages`.
Otherwise it is not so important. In this case it could be in `pythonPackages` with an alias in `all-packages.nix`

Reply:
They do support multiple versions of python. So I've done as @Mic92 suggested and aliased.